### ISSUE TITLE
Is get_trace_tool_link() still usable?

### ISSUE THREAD

**Issue by adrienpacifico** [openfisca/openfisca-core#736]
Is get_trace_tool_link() still usable ?
https://github.com/openfisca/openfisca-core/blob/e3e2118685baa34a12600963bb355d428d7099dc/openfisca_core/tools/__init__.py#L68

### Here is what I did:
```
from openfisca_core import tools
print(tools.get_trace_tool_link(scenario, variables = ['irpp'],
                                                   api_url = 'http://api.openfisca.fr', 
                                                   trace_tool_url =  'http://www.openfisca.fr/tools/trace'))
```
### Here is what I expected to happen:
Get a web-browser that gives me trace of the computed variable.

### Here is what actually happened:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-14-5c10be8a06ed> in <module>()
      1 from openfisca_core import tools
----> 2 print(tools.get_trace_tool_link(scenario, variables = ['irpp'], api_url = 'http://api.openfisca.fr', trace_tool_url =  'http://www.openfisca.fr/tools/trace'))

~/.local/share/virtualenvs/Python_3_7_0/lib/python3.7/site-packages/openfisca_core/tools/__init__.py in get_trace_tool_link(scenario, variables, api_url, trace_tool_url)
     75         'variables': variables,
     76         }
---> 77     url = trace_tool_url + '?' + urllib.urlencode({
     78         'simulation': json.dumps(simulation_json),
     79         'api_url': api_url,

AttributeError: module 'urllib' has no attribute 'urlencode'
````

Ok, It is probably a Python 3 vs 2 issue, but would it still work after correction?

In other words, is get_trace_tool_link() still a thing?

Best

**Comment by fpagnoux** [comment]
Hi @adrienpacifico,

No, we now have a different trace tool, and there is not yet a graphical way of visualizing it.

- When you create a simulation, just add `trace = True` to either the `Simulation` constructor, or the `new_simulation()` scenario method.
- Then check the content of `simulation.tracer.trace`

It gives you information about the dependencies of all computed variables

**Comment by fpagnoux** [comment]
Closing the issue, as the question has been answered. 
Feel free to ask for help if you need more info about the new trace though.

**Comment by fpagnoux** [comment]
Actually, @guillett and @alexsegura have been working on a new graphical interface (POC): https://betagouv.github.io/openfisca-tracer/

**Comment by adrienpacifico** [comment]
Thank you @fpagnoux ,

That's very clear.

I just saw your answer, I need to learn to manage GitHub notifications more efficiently...

One comment and 2 questions if I may: 
`get_trace_tool_link()` was working pretty well for me and it was fitting my needs. 
1. Why a new POC tracer when we had one that was working?
2. Where can I find the discussions that lead to launching that new POC tracer?
